ngrok/assets/policies.json

```diff
@@ -0,0 +1,33 @@
+{
+  "policies": {
```

Comment:
We are renaming top-level configuration to be singular 'policy' 

Internally and externally, the elements in the inbound and outbound arrays will be referred to as "Policy Rules" 
Issue here for rename on agent protocol https://github.com/ngrok-private/ngrok/issues/25765